client/state/current-user/selectors.js

```diff
@@ -35,12 +30,7 @@ export function isUserLoggedIn( state ) {
  * @returns {?object}        Current user
  */
 export function getCurrentUser( state ) {
-	const userId = getCurrentUserId( state );
-	if ( ! userId ) {
-		return null;
-	}
-
-	return getUser( state, userId );
+	return get( state, [ 'currentUser', 'user' ], null );
```

Comment:
Or perhaps we can completely remove the `get` usage in favor of a `state.currentUser.user ?? null` or even `state.currentUser.user` as it will default to `null` anyway?

Reply:
I plan to remove Lodash from this module in a separate, mono-thematic PR 🙂

Reply:
Sounds good to me 👍